bin/modes.js

```diff
@@ -60,7 +62,8 @@ function init(argv){
 
   var options = {
     showWarnings: true,
-    output: "tests"
+    output: "tests",
+    configPath: false
```

Comment:
I think I would prefer to set the default value hear instead of https://github.com/stoeffel/elm-doc-test/pull/16/files#diff-c8bf800c66995a617bab8621d27607ccR62